README.md

```diff
@@ -1,199 +1,240 @@
-sourced (`max_lines_to_paste`).
+- `R_source` and `after_R_start` have been replaced with more powerful `hook`
+    options.
 
-The options for displaying R documentation (`nvimpager`) are now: `"split_h"`,
-`"split_v"`, `"tab"`, `"float"` (not implemented yet), and `"no"`.
+- `nvimpager`, which controls how R documentation is displayed, now has possible
+    options `"split_h"`, `"split_v"`, `"tab"`, `"float"` (not implemented yet),
+    and `"no"`.
 
-The options `openpdf` and `openhtml` were renamed as `open_pdf` and
-`open_html`, they now are strings and there is a minor change in how they
-behave.
+- `open_pdf` replaces `openpdf` and `openhtml`; see the documentation for
+    details.
 
-The option `nvim_wd` was renamed as `setwd` and it now is a string and its
-default value is "no".
+- `setwd` replaces `nvim_wd`. The new default value is `"no"`.
 
-The option `notmuxconf` was renamed as `config_tmux` to avoid the negation of
-the negation `notmuxconf=false` or the even more awkward confirmation of the
-negation `notmuxconf=true`. The default value of `config_tmux` is `true`.
+- `config_tmux` replaces `notmuxconf`. The new default value is `true`.
 
-The `:RFormat` command no longer has the option to use the package `formatR`;
-now, the `styler` package is required.
+- `:RFormat` now requires {styler} to be installed; {formatR} is no longer
+    supported.
 
 ### New features
 
-There are two new commands:
+#### New commands
 
 - `:RMapsDesc` displays the list of key bindings followed by short
   descriptions.
 
 - `:RConfigShow` displays the list of configuration options and their current
   values.
 
-There is one new command to send the above-piped chain of commands. Its
-default key binding is `<LocalLeader>sc`.
+#### New keybindings
 
-There is a new command to install missing packages detected by the
-[languageserver]: `<LocalLeader>ip`.
+- `<LocalLeader>sc` sends a piped chain of commands.
 
-There is a new option: `auto_quit`.
+- `<LocalLeader>ip` installs missing packages detected by
+    [languageserver](https://github.com/REditorSupport/languageserver).
 
-If you have [colorout] installed, and if you are not loading it in your
-`~/.Rprofile`, it should be version `1.3-1` or higher. Reason: R.nvim calls
-the function `colorout::isColorOut()` which in previous `colorout` versions
-were unduly enabling the output colorizing.
+- `<LocalLeader>sp` splits a filepath under the cursor into individual
+    components concatenated using `paste()` (requires
+    [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) to be
+    installed).
 
-There are two new commands available to separate a file path into its different
-components.
+- `<LocalLeader>sh` splits a filepath under the cursor into individual 
+    components concatenated using `here::here()` (requires
+    [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) to be
+    installed).
```

Comment:
This is now only 1 function `<localleader>sp`. The split function to use can be configured with `path_split_fun` which can be 

```lua
  path_split_fun   = { "here::here", "here", "file.path", "fs::path", "path" }
```

Reply:
Fixed!